admin/spec/features/accounts_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe "Accounts", type: :feature do
+  let(:user) { create(:admin_user, email: 'admin@example.com') }
+
+  before do
+    allow_any_instance_of(SolidusAdmin::BaseController).to receive(:spree_current_user).and_return(user)
```

Comment:
I think eventually we'll need a helper method for doing the sign in programmatically, but for now this is ok 👍